assets/javascripts/kitten/components/organisms/header-nav/stories/kisskissbankbank-again.js

```diff
@@ -0,0 +1,354 @@
+          </HeaderNav.Hidden>
+
+          <HeaderNav.Hidden max="xxs">
+            <KissKissBankBankLogo />
+          </HeaderNav.Hidden>
+        </HeaderNav.Logo>
+
+         <HeaderNav.Hidden max="m">
+          <HeaderNav.Nav>
+            <Navigation />
+          </HeaderNav.Nav>
+        </HeaderNav.Hidden>
+
+        <HeaderNav.Right padded>
+          <SearchInput />
+  
+          <HeaderNav.Logged>
+            <HeaderNav.UserMenuNext dropdownAnchorSide="right" padding={false}>
+              <HeaderNav.UserMenuNext.Button
+                hasArrow
+                backgroundColor={COLORS.background1}
+                backgroundColorHover="transparent"
+                backgroundColorActive="transparent"
+                color={COLORS.font1}
+                colorHover={COLORS.primary1}
+                colorActive={COLORS.font1}
+              >
+                <AvatarWithTextAndBadge className="k-u-margin-left-single@xs-down">
+                  <AvatarWithTextAndBadge.Image src="/kitten.jpg">
+                    <AvatarWithTextAndBadge.Badge>
+                      2
+                    </AvatarWithTextAndBadge.Badge>
+                  </AvatarWithTextAndBadge.Image>
+
+                  <AvatarWithTextAndBadge.Text
+                    className="k-u-hidden@xs-down"
+                    withEllipsisOverflow={true}
+                  >
+                    <Text lineHeight="normal" weight="regular">
+                      Jean Charles Édouard
+                    </Text>
+                  </AvatarWithTextAndBadge.Text>
+                </AvatarWithTextAndBadge>
+              </HeaderNav.UserMenuNext.Button>
+              <HeaderNav.UserMenuNext.Navigation>
+                <InnerUserMenu />
+              </HeaderNav.UserMenuNext.Navigation>
+            </HeaderNav.UserMenuNext>
+          </HeaderNav.Logged>
+
+          {/* <HeaderNav.Hidden max="m">
+            <Button
+              icon
+              size="tiny"
+              rounded
+              className="k-u-margin-right-single"
+            >
+              <GiftIcon width={15} height={16} />
+            </Button>
+          </HeaderNav.Hidden> */}
+          
+          <HeaderNav.Hidden max="m">
+            <Button
+              size="tiny"
+              borderRadius={100}
+              className="k-u-margin-right-single"
+              style={{ minWidth: 'auto', paddingLeft: '18px', paddingRight: '18px', borderRadius:'100px'}}
+            >
+              Blog
+            </Button>
+          </HeaderNav.Hidden>
+
+          <HeaderNav.Hidden max="m">
+            <Button
+              icon
+              size="tiny"
+              rounded
+              className="k-u-margin-right-single"
+            >
+              <EnvelopeIcon />
+            </Button>
+          </HeaderNav.Hidden>
+
+          <HeaderNav.LoggedOut>
+
+          <HeaderNav.Hidden max="m"> 
+            <HeaderNav.Nav.Item href="#">
```

Comment:
par défaut le `HeaderNav.Nav.Item` est un `li`, sauf que là il est contenu dans un 

```suggestion
            <HeaderNav.Nav.Item href="#" as="div" className="k-u-alignSelf-stretch">
```